### DISCUSSION TITLE
When using @sendinblue/client npm package deployments with adapter-cloudflare fail

### DISCUSSION THREAD

**Discussion by fev4** [sveltejs/kit#9499] [Q&A]
Hey there,

After I installed `@sendinblue/client` and trying to deploy to Cloudflare with `adapter-cloudflare` I'm getting the following errors. This error appears when running `pnpm build`. When I uninstall the `@sendinblue/client` package and then run `pnpm build` again, it works correctly.

What can I try to fix this?

<details>
<summary>
Error log
</summary>
The package "url" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "url"

    node_modules/.pnpm/request@2.88.2/node_modules/request/lib/tunnel.js:3:18:
      3 │ var url = require('url')
        ╵                   ~~~~~

  The package "url" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "net"

    node_modules/.pnpm/tunnel-agent@0.6.0/node_modules/tunnel-agent/index.js:3:18:
      3 │ var net = require('net')
        ╵                   ~~~~~

  The package "net" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "tls"

    node_modules/.pnpm/tunnel-agent@0.6.0/node_modules/tunnel-agent/index.js:4:18:
      4 │   , tls = require('tls')
        ╵                   ~~~~~

  The package "tls" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "http"

    node_modules/.pnpm/tunnel-agent@0.6.0/node_modules/tunnel-agent/index.js:5:19:
      5 │   , http = require('http')
        ╵                    ~~~~~~

  The package "http" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "https"

    node_modules/.pnpm/tunnel-agent@0.6.0/node_modules/tunnel-agent/index.js:6:20:
      6 │   , https = require('https')
        ╵                     ~~~~~~~

  The package "https" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "events"

    node_modules/.pnpm/tunnel-agent@0.6.0/node_modules/tunnel-agent/index.js:7:21:
      7 │   , events = require('events')
        ╵                      ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "assert"

    node_modules/.pnpm/tunnel-agent@0.6.0/node_modules/tunnel-agent/index.js:8:21:
      8 │   , assert = require('assert')
        ╵                      ~~~~~~~~

  The package "assert" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/tunnel-agent@0.6.0/node_modules/tunnel-agent/index.js:9:19:
      9 │   , util = require('util')
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/forever-agent@0.6.1/node_modules/forever-agent/index.js:4:19:
      4 │ var util = require('util')
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "http"

    node_modules/.pnpm/forever-agent@0.6.1/node_modules/forever-agent/index.js:5:20:
      5 │   , Agent = require('http').Agent
        ╵                     ~~~~~~

  The package "http" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "net"

    node_modules/.pnpm/forever-agent@0.6.1/node_modules/forever-agent/index.js:6:18:
      6 │   , net = require('net')
        ╵                   ~~~~~

  The package "net" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "tls"

    node_modules/.pnpm/forever-agent@0.6.1/node_modules/forever-agent/index.js:7:18:
      7 │   , tls = require('tls')
        ╵                   ~~~~~

  The package "tls" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "https"

    node_modules/.pnpm/forever-agent@0.6.1/node_modules/forever-agent/index.js:8:23:
      8 │   , AgentSSL = require('https').Agent
        ╵                        ~~~~~~~

  The package "https" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "url"

    node_modules/.pnpm/request@2.88.2/node_modules/request/lib/oauth.js:3:18:
      3 │ var url = require('url')
        ╵                   ~~~~~

  The package "url" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/http-signature@1.2.0/node_modules/http-signature/lib/utils.js:5:19:
      5 │ var util = require('util');
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/http-signature@1.2.0/node_modules/http-signature/lib/signer.js:4:21:
      4 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "http"

    node_modules/.pnpm/http-signature@1.2.0/node_modules/http-signature/lib/signer.js:5:19:
      5 │ var http = require('http');
        ╵                    ~~~~~~

  The package "http" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/errors.js:4:19:
      4 │ var util = require('util');
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "querystring"

    node_modules/.pnpm/request@2.88.2/node_modules/request/lib/querystring.js:4:26:
      4 │ var querystring = require('querystring')
        │                           ~~~~~~~~~~~~~
        ╵                           "./querystring"

  The package "querystring" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "stream"

    node_modules/.pnpm/isstream@0.1.2/node_modules/isstream/isstream.js:1:21:
      1 │ var stream = require('stream')
        ╵                      ~~~~~~~~

  The package "stream" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/request@2.88.2/node_modules/request/lib/hawk.js:3:21:
      3 │ var crypto = require('crypto')
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "url"

    node_modules/.pnpm/aws4@1.12.0/node_modules/aws4/aws4.js:2:18:
      2 │     url = require('url'),
        ╵                   ~~~~~

  The package "url" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "fs"

    node_modules/.pnpm/request@2.88.2/node_modules/request/lib/har.js:3:17:
      3 │ var fs = require('fs')
        ╵                  ~~~~

  The package "fs" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/combined-stream@1.0.8/node_modules/combined-stream/lib/combined_stream.js:1:19:
      1 │ var util = require('util');
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/http-signature@1.2.0/node_modules/http-signature/lib/signer.js:6:19:
      6 │ var util = require('util');
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/aws-sign2@0.7.0/node_modules/aws-sign2/index.js:22:21:
      22 │ var crypto = require('crypto')
         ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/http-signature@1.2.0/node_modules/http-signature/lib/verify.js:4:21:
      4 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "stream"

    node_modules/.pnpm/combined-stream@1.0.8/node_modules/combined-stream/lib/combined_stream.js:2:21:
      2 │ var Stream = require('stream').Stream;
        ╵                      ~~~~~~~~

  The package "stream" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/signature.js:8:21:
      8 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/identity.js:7:21:
      7 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/identity.js:11:19:
      11 │ var util = require('util');
         ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "querystring"

    node_modules/.pnpm/aws4@1.12.0/node_modules/aws4/aws4.js:3:26:
      3 │     querystring = require('querystring'),
        ╵                           ~~~~~~~~~~~~~

  The package "querystring" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/aws4@1.12.0/node_modules/aws4/aws4.js:4:21:
      4 │     crypto = require('crypto'),
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "querystring"

    node_modules/.pnpm/request@2.88.2/node_modules/request/lib/har.js:4:17:
      4 │ var qs = require('querystring')
        │                  ~~~~~~~~~~~~~
        ╵                  "./querystring"

  The package "querystring" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/utils.js:28:21:
      28 │ var crypto = require('crypto');
         ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "url"

    node_modules/.pnpm/aws-sign2@0.7.0/node_modules/aws-sign2/index.js:23:20:
      23 │   , parse = require('url').parse
         ╵                     ~~~~~

  The package "url" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/http-signature@1.2.0/node_modules/http-signature/lib/parser.js:4:19:
      4 │ var util = require('util');
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/private-key.js:8:21:
      8 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "assert"

    node_modules/.pnpm/assert-plus@1.0.0/node_modules/assert-plus/assert.js:4:21:
      4 │ var assert = require('assert');
        │                      ~~~~~~~~
        ╵                      "./assert"

  The package "assert" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "stream"

    node_modules/.pnpm/assert-plus@1.0.0/node_modules/assert-plus/assert.js:5:21:
      5 │ var Stream = require('stream').Stream;
        ╵                      ~~~~~~~~

  The package "stream" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "assert"

    node_modules/.pnpm/asn1@0.2.6/node_modules/asn1/lib/ber/writer.js:3:21:
      3 │ var assert = require('assert');
        ╵                      ~~~~~~~~

  The package "assert" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/certificate.js:8:21:
      8 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/certificate.js:12:19:
      12 │ var util = require('util');
         ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "buffer"

    node_modules/.pnpm/safer-buffer@2.1.2/node_modules/safer-buffer/safer.js:5:21:
      5 │ var buffer = require('buffer')
        ╵                      ~~~~~~~~

  The package "buffer" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/private-key.js:12:19:
      12 │ var util = require('util');
         ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "assert"

    node_modules/.pnpm/asn1@0.2.6/node_modules/asn1/lib/ber/reader.js:3:21:
      3 │ var assert = require('assert');
        ╵                      ~~~~~~~~

  The package "assert" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "stream"

    node_modules/.pnpm/delayed-stream@1.0.0/node_modules/delayed-stream/lib/delayed_stream.js:1:21:
      1 │ var Stream = require('stream').Stream;
        ╵                      ~~~~~~~~

  The package "stream" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/delayed-stream@1.0.0/node_modules/delayed-stream/lib/delayed_stream.js:2:19:
      2 │ var util = require('util');
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/fingerprint.js:8:21:
      8 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/assert-plus@1.0.0/node_modules/assert-plus/assert.js:6:19:
      6 │ var util = require('util');
        ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/key.js:7:21:
      7 │ var crypto = require('crypto');
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/request@2.88.2/node_modules/request/lib/oauth.js:8:21:
      8 │ var crypto = require('crypto')
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "stream"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/ed-compat.js:9:21:
      9 │ var stream = require('stream');
        ╵                      ~~~~~~~~

  The package "stream" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/ed-compat.js:10:19:
      10 │ var util = require('util');
         ╵                    ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/formats/putty.js:13:21:
      13 │ var crypto = require('crypto');
         ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/jsprim@1.4.2/node_modules/jsprim/lib/jsprim.js:6:23:
      6 │ var mod_util = require('util');
        ╵                        ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/oauth-sign@0.9.0/node_modules/oauth-sign/index.js:1:21:
      1 │ var crypto = require('crypto')
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "assert"

    node_modules/.pnpm/extsprintf@1.3.0/node_modules/extsprintf/lib/extsprintf.js:5:25:
      5 │ var mod_assert = require('assert');
        ╵                          ~~~~~~~~

  The package "assert" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/verror@1.10.0/node_modules/verror/lib/verror.js:6:23:
      6 │ var mod_util = require('util');
        ╵                        ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/formats/openssh-cert.js:17:21:
      17 │ var crypto = require('crypto');
         ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/formats/pem.js:10:21:
      10 │ var crypto = require('crypto');
         ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/formats/ssh-private.js:14:21:
      14 │ var crypto = require('crypto');
         ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "util"

    node_modules/.pnpm/extsprintf@1.3.0/node_modules/extsprintf/lib/extsprintf.js:6:23:
      6 │ var mod_util = require('util');
        ╵                        ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying
  to bundle for node? You can use "platform: 'node'" to do that, which will remove this
  error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/sshpk@1.17.0/node_modules/sshpk/lib/dhe.js:10:21:
      10 │ var crypto = require('crypto');
         ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

✘ [ERROR] Could not resolve "crypto"

    node_modules/.pnpm/ecc-jsbn@0.1.2/node_modules/ecc-jsbn/index.js:1:21:
      1 │ var crypto = require("crypto");
        ╵                      ~~~~~~~~

  The package "crypto" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.

error during build:
Error: Build failed with 79 errors:
node_modules/.pnpm/asn1@0.2.6/node_modules/asn1/lib/ber/reader.js:3:21: ERROR: Could not resolve "assert"
node_modules/.pnpm/asn1@0.2.6/node_modules/asn1/lib/ber/writer.js:3:21: ERROR: Could not resolve "assert"
node_modules/.pnpm/assert-plus@1.0.0/node_modules/assert-plus/assert.js:4:21: ERROR: Could not resolve "assert"
node_modules/.pnpm/assert-plus@1.0.0/node_modules/assert-plus/assert.js:5:21: ERROR: Could not resolve "stream"
node_modules/.pnpm/assert-plus@1.0.0/node_modules/assert-plus/assert.js:6:19: ERROR: Could not resolve "util"
...
    at failureErrorWithLog (/home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:1604:15)
    at /home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:1056:28
    at /home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:1001:67
    at buildResponseToResult (/home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:1054:7)
    at /home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:1166:14
    at responseCallbacks.<computed> (/home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:701:9)
    at handleIncomingPacket (/home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:756:9)
    at Socket.readFromStdout (/home/feva/projects/zentu/dev/node_modules/.pnpm/esbuild@0.16.17/node_modules/esbuild/lib/main.js:677:7)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
 ELIFECYCLE  Command failed with exit code 1.
</details>

**Comment by nosovk** [comment]
It seems that the problem is not related to SvelteKit. It seems that https://github.com/sendinblue/APIv3-typescript-library is not compatible with Cloudflare workers environment. You may ask them to update the library to be runnable in web-worker environment (Cloudflare workers is not NodeJS, it's more like web-workers in browser), or just make http calls to api directly.
https://api.sendinblue.com/v3/swagger_definition.yml here you can find swagger file that covers sendinblue api.
